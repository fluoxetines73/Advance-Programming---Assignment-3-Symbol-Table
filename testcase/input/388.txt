BEGIN
PRINT
RPRINT
PRINT
RPRINT
INSERT c number
INSERT x string
INSERT f_FF string
BEGIN
BEGIN
ASSIGN f_FF 'HelloWorld'
RPRINT
INSERT 21_3c 123
LOOKUP x random
END
END
END