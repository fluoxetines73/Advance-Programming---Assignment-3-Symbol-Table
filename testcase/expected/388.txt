Invalid: INSERT 21_3c 123